app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def show
+    @user = current_user
```

Comment:
You can actually just refer to `current_user` in the show.erb file that this links to. If you're looking for a user in the database, I would recommend using `User.find(user_id: current_user.id)` or something similar.